### DISCUSSION TITLE
Free as freedom

### DISCUSSION THREAD

**Discussion by Swinec** [alvr-org/ALVR#952] [Q&A] [answered]
Not as free beer.

Why MIT-license?
Under the GPL2 license, the project will be more independent, it will not be able to be stolen by bloat corporations and chines plagiarists.
Why take care on a project that can be stolen from you at any moment without investing anything in the project?
MIT is a free as free beer, not as freedom. 
ALVR may become an important part of linux userspace ecosystem.

**Comment by zmerp** [comment, marked as answer]
Freedom is a very relative term. I personally see freedom as in anyone (both users or businesses) can benefit from the project. I welcome any company to take benefit from ALVR codebase, with just the MIT enforced rule to provide a copy of the license. If the company that uses ALVR code respects the license, the user would be notified that some of ALVR code were used and they can make a conscious decision to use the original ALVR project itself.

That being said, it's impossible to change the license. This is a fork of polygraphene's repository, who chose the license. Many many people contributed to the project then and since then, and it's unlikely we can make a coordinated decision with all.
